hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ReferenceObjects.java

```diff
@@ -229,6 +229,9 @@ private static boolean isEqualStackTrace(StackTraceElement stackTraceElement1, S
     public static Xid anXid = new SerializableXID(1, aString.getBytes(), aString.getBytes());
     public static List<Map.Entry<Data, Data>> aListOfEntry = Collections.<Map.Entry<Data, Data>>singletonList(
             new AbstractMap.SimpleEntry<Data, Data>(aData, aData));
+    public static List<Map.Entry<String, byte[]>> aListOfStringToByteArrEntry
+            = Collections.<Map.Entry<String, byte[]>>singletonList(
```

Comment:
I saw the same thing above with `aListOfEntry` so I figured this was enough. Is it?